bin/minifollowups/pycbc_page_snglinfo

```diff
@@ -84,24 +88,69 @@ else:
 # make a table for the single detector information ############################
 time = sngl_file.end_time
 utc = lal.GPSToUTC(int(time))[0:6]
-data = [[pycbc.results.dq.get_summary_page_link(args.instrument, utc),
-        str(datetime.datetime(*utc)),
-        '%.3f'  % time,
-        '%5.2f' % sngl_file.snr,
-        '%5.2f' % stat,
-        '%5.2f' % sngl_file.rchisq,
-        '%i'    % sngl_file.get_column('chisq_dof'),
-        '%5.2f' % sngl_file.get_column('coa_phase'),
-        '%5.2f' % sngl_file.mass1,
-        '%5.2f' % sngl_file.mass2,
-        '%5.2f' % sngl_file.mchirp,
-        '%5.2f' % sngl_file.spin1z,
-        '%5.2f' % sngl_file.spin2z,
-        '%5.2f' % sngl_file.template_duration,
-       ]]
+
+try:
+    psd_var = sngl_file.psd_var_val
+except:
+    psd_var = None
+
+try:
+    sg_chisq = sngl_file.sgchisq
+except:
+    sg_chisq = None
+
+# This might involve a lot of lines, but it's clear what goes in the output
+# and in what order.
```

Comment:
this comment is maybe what you say to yourself while coding, but doesn't explain the code. 
instead, say what info 'headers' and 'data' are going to hold.

Reply:
Changed as requested.